src/content/homepage/charts/second-chart/convert-msaxlsx-to-html.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+if [ $# -ne 1 ]; then
+     echo "usage: $0 <xlsx-filename>"
+     exit 1
+fi
+
+# https://github.com/dilshod/xlsx2csv (this gracefully handles unicode nastiness, thankfully).
+pip install xlsx2csv
```

Comment:
yup that should be easy enough. (I wrote this as more of a quick hack that folks can use once/year, than to be robust extensible code, but these are easy enough changes).